### ISSUE TITLE
Restart option does not restart app

### ISSUE THREAD

**Issue by minter** [electron/update-electron-app#57]
I'm working with update.electronjs.org, and pushing out new releases. I'm getting notified that there is a new release available, with the options of `Later` and `Restart`. However, choosing `Restart` does nothing.

![Restart doesn't](https://user-images.githubusercontent.com/955/81357961-171bfa80-90a3-11ea-852a-a532521f4f91.gif)

Here's the output of my ShipIt stderr log:
```
2020-05-07 20:34:33.545 ShipIt[80622:8057145] Beginning installation
2020-05-07 20:34:37.897 ShipIt[80622:8056127] Moved bundle from file:///Applications/Mx.%20Voice.app/ to file:///var/folders/kr/h88rnk_d4b551p0fmqgg074w0000gn/T/com.electron.mx.-voice.ShipIt.Zk5twCOs/Mx.%20Voice.app
2020-05-07 20:34:37.898 ShipIt[80622:8056127] Moved bundle from file:///var/folders/kr/h88rnk_d4b551p0fmqgg074w0000gn/T/com.electron.mx.-voice.ShipIt.siNxUhc7/Mx.%20Voice.app to file:///Applications/Mx.%20Voice.app/
2020-05-07 20:34:38.513 ShipIt[80622:8056127] Couldn't remove owned bundle at location file:///var/folders/kr/h88rnk_d4b551p0fmqgg074w0000gn/T/com.electron.mx.-voice.ShipIt.siNxUhc7/Mx.%20Voice.app, error NSError { domain: NSCocoaErrorDomain, code: 4, description: "“Mx. Voice.app” couldn’t be removed.", failureReason: "The file doesn’t exist.", underlying error: NSError { domain: NSPOSIXErrorDomain, code: 2, description: "The operation couldn’t be completed. No such file or directory" }, userInfo: {
    NSFilePath = "/var/folders/kr/h88rnk_d4b551p0fmqgg074w0000gn/T/com.electron.mx.-voice.ShipIt.siNxUhc7/Mx. Voice.app";
    NSUserStringVariant =     (
        Remove
    );
} }
2020-05-07 20:34:39.576 ShipIt[80622:8056127] Installation completed successfully
```

There's nothing in the ShipIt stdout file.

Any ideas what might be going on?

**Comment by mahnunchik** [comment]
I've faced with the same behaviour.

**Comment by mahnunchik** [comment]
It seems the `showMessageBox` method has outdated signature: https://github.com/electron/update-electron-app/blob/master/index.js#L85-L87

https://www.electronjs.org/docs/api/dialog#dialogshowmessageboxbrowserwindow-options

**Comment by rlingineni** [comment]
Can repro the same issue. Is there a fix for the above? Not exactly sure I follow the outdated signature.

**Comment by rlingineni** [comment]
Thanks @HashimotoYT! Will this automatically be deployed to NPM so I can download the latest update

**Comment by erickzhao** [comment]
Hey @rlingineni, sorry for the delay, but the package has now been updated: https://github.com/electron/update-electron-app/releases/tag/v2.0.1

**Comment by felipap** [comment]
I'm also seeing this. Has this been fixed?

**Comment by erickzhao** [comment]
@felipap The issue is fixed if you're using v2.0.1 or above of this package. If you're experiencing this in a recent version and are able to isolate the example to a minimal repro, please file a new issue for maintainers to triage.
